<launch>
	<!-- segmentation service -->
	<!--<arg name="tabletop_segmentation_service_name" default="/extract_objects_table" />-->

	<!-- topic where the hypotheses go -->
	<arg name="bottomup_detector_hypotheses_out" default="object_detector" />

	<!-- topic where the markers go -->
	<arg name="detector_markers_out" default="detector_markers" />

	<node pkg="objectrecognition" name="object_detector" type="object_detector_sv">
		<!--database parameters -->
		<rosparam command="load" file="$(find objectrecognition)/config/db.yaml"/>
		<rosparam command="load" file="$(find objectrecognition)/config/parameters.yaml"/>

		<remap from="bottomup_detector_hypotheses_out" to="$(arg bottomup_detector_hypotheses_out)" />
		<remap from="detector_markers_out" to="$(arg detector_markers_out)" />
		<!--<remap from="tabletop_segmentation" to="$(arg tabletop_segmentation_service_name)" />-->
	</node> 
</launch>


